bdk_chain/src/chain_graph.rs

```diff
@@ -173,22 +173,20 @@ impl<P: ChainPosition> ChainGraph<P> {
         Ok(changeset)
     }
 
-    /// Given a transaction, return an iterator of in-chain [`Txid`]s that conflict with it (spends
-    /// at least one of the same inputs).
+    /// Given a transaction, return an iterator of `txid`s that conflict with it (spends at least
+    /// one of the same inputs). This includes all descendants of conflicting transactions.
     ///
-    /// This method is comparable to [`TxGraph::conflicting_txids()`] which returns all conflicting
-    /// transactions, whereas this method only returns conflicts that exist in the [`SparseChain`].
-    pub fn conflicting_txids_in_chain<'a>(
+    /// This method only returns conflicts that exist in the [`SparseChain`] as transactions that
+    /// are not included in [`SparseChain`] are already considered as evicted.
+    pub fn tx_conflicts_in_chain<'a>(
```

Comment:
We should have a test for this that evicts descendants.

Reply:
https://github.com/LLFourn/bdk_core_staging/blob/842cfc8b50e1eca5b1d83b598eb3d2e4ae2147ca/bdk_chain/tests/test_chain_graph.rs#L350-L354

We currently do